app/services/reporting/account_deletion_rate_report.rb

```diff
@@ -27,22 +27,21 @@ def account_deletion_emailable_report
 
     def deleted_user_count
       @deleted_user_count ||= Reports::BaseReport.transaction_with_timeout do
-        DeletedUser.where(user_created_at: start_date..end_date).count
+        DeletedUser.
+          where(deleted_at: start_date..end_date).
+          where('user_created_at < ?', end_date).
+          count
       end
     end
 
-    def user_count
-      @user_count ||= Reports::BaseReport.transaction_with_timeout do
-        User.where(created_at: start_date..end_date).count
+    def fully_registered_users
+      @fully_registered_users ||= Reports::BaseReport.transaction_with_timeout do
+        RegistrationLog.where(registered_at: start_date..end_date).count
       end
     end
 
-    def users_and_deleted_for_period
-      deleted_user_count + user_count
-    end
-
     def deletion_rate
-      deleted_user_count.to_f / users_and_deleted_for_period.to_f
+      deleted_user_count.to_f / fully_registered_users.to_f
     end
```

Comment:
One thing to note, I'm not 100% sure I agree with [the calculation on data.login.gov](https://github.com/18F/identity-reporting/blob/49618b22f417bbc3ef1221ac38f53f18fd86703f/src/components/account-deletions-report.tsx#L98), because "fully registered users" is not safe for backdating, the registrations logs rows are deleted and not recoverable, where the "total registered" is, we can approximate counts far back in time based on `deleted_users`

Reply:
For what it's worth, I'm not attached to that logic (as the person who wrote it), if we want to update to something more accurate and backport to data.login.gov .

But also we're tentatively considering to want to change this to an "Account Reset Rate" anyways, so maybe it's best to hold off for that.